### DISCUSSION TITLE
SONOFF RF Bridge R2 e Tasmota

### DISCUSSION THREAD

**Discussion by array81** [indomus/forum#1890] [Hardware & Firmware]
Sarei interessato a comprare il SONOFF RF Bridge. Ho visto che si trova la versione R2 e mi piacerebbe comprare quest'ultima, prima di farlo vorrei capire se posso avere problemi a flashare il firmware. Leggendo su interne infatti la cosa non mi è chiarissima. Ho trovato un forum in cui si sosteneva che la versione R2 (da quanto ho capito solo nella revisione più recente, quindi non in tutte), è sempre compatibile con Tasmota ma ha problemi con Portisch (ovvero con il firmware modificato della componente radio che comunque aggiunge funzionalità e non è essenziale).
Avete riscontro in merito?

**Comment by snakuzzo** [comment]
In realtà [QUI](https://tasmota.github.io/docs/devices/Sonoff-RF-Bridge-433/) dovrebbe esserci tutto quello che ti serve

**Comment by vincenzocoletta** [comment]
Ciao, io mi sto cimentando ora e non ho ancora trovato una guida affidabile e precisa (viste le mie basiche conoscenze in merito mi serve un tutorial passopasso). Tu ci sei riuscito, hai qualche indicazione da suggerirmi? Il mio HW è R2 v2.2…
